### ISSUE TITLE
(DSNCORE-1346) Chips as buttons

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#2016]
https://jira.tid.es/browse/DSNCORE-1346

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket

**Comment by aweell** [comment]
> Acceptance Criteria (numbered):
> 
> AC1. Header chips must be read as buttons by the screen reader. A confirmar este comportamiento

The chips when used with onPress are a button by default, this can be changed also using the `role` property. This criteria should be already covered by the actual component.

> The 'Tienes facturas pendientes' button should not be announced as part of a list when it doesn't look like a list.

The chip component is not aware of the context is being used into. If the button chips are included in a list and they shouldn't this is something that is defined in the product.
